modules/recommended-etcd-practices.adoc

```diff
@@ -34,9 +40,6 @@ IMPORTANT: Avoid using the Network File System (NFS) protocol.
 
 Some key metrics to monitor on a deployed {product-title} cluster are p99 of etcd disk write ahead log duration and the number of etcd leader changes. Use Prometheus to track these metrics.
 
-* The `etcd_disk_wal_fsync_duration_seconds_bucket` metric reports the etcd disk fsync duration.
-* The `etcd_server_leader_changes_seen_total` metric reports the leader changes.
```

Comment:
Rather than remove these, would we add them to the "Monitor these key metrics" section above?

Reply:
Hey @afcollins, I've done to the PR. PTAL

Reply:
thanks. LGTM